v3/src/components/graph/utilities/graph-utils.ts

```diff
@@ -157,16 +157,22 @@ export function lineToAxisIntercepts(iSlope: number, iIntercept: number,
   }
 }
 
-export function equationString(slope: number, intercept: number, attrNames: {x: string, y: string}) {
-  const float = format('.4~r')
+export function equationString(
+  slope: number, intercept: number, attrNames: {x: string, y: string}, sumOfSquares?: number
+) {
+  const float = format(".3~r")
+  const floatSumOfSquares = format(",.0f")
```

Comment:
Is this what v2 uses? Seems like there could be data sets with very small residuals where this wouldn't work well.

Reply:
@kswenson This isn't exactly what V2 uses. The use of D3's `format` was something of a placeholder when it was first introduced in graph-utils.ts. I looked into this a bit and discussed it with @bfinzer. We determined it'd be best to make matching V2's formatting a separate piece of work. I added [a story about that](https://www.pivotaltracker.com/story/show/186611743) to PT.